db-repo/include/emeralddb/core.hpp

```diff
@@ -0,0 +1,72 @@
+/*    Copyright 2009 10gen Inc.
```

Comment:
in core.hpp add macro variables, common cpp header files and error numbers for globa/context modules include